provisio-maven-plugin/src/main/java/ca/vanzyl/maven/plugins/provisio/GeneratorMojo.java

```diff
@@ -33,19 +34,19 @@
 import java.io.File;
 import java.io.IOException;
 import java.io.Writer;
+import java.nio.file.Files;
 import java.util.ArrayList;
-import java.util.Comparator;
 import java.util.List;
 
-@Mojo(name = "generateDependencies", requiresDependencyResolution = ResolutionScope.COMPILE_PLUS_RUNTIME)
+@Mojo(name = "generateDependencies", defaultPhase = LifecyclePhase.GENERATE_RESOURCES, requiresDependencyResolution = ResolutionScope.COMPILE_PLUS_RUNTIME)
```

Comment:
Adding the `defaultPhase` here does _not_ turn it on by default. I'm not sure why. I think it should be on by default.

Reply:
You need to register the Mojo in components.xml. See https://github.com/jvanzyl/provisio/blob/master/provisio-maven-plugin/src/main/resources-filtered/META-INF/plexus/components.xml.

Reply:
It works!

@jvanzyl I saw you merged master here to resolve conflicts but I also forced pushed my latest changes. Tests are passing. I replaced JUnit with some Scala lib in tests to avoid conflicts since Provisio actually uses JUnit.